src/driver.rs

```diff
@@ -292,13 +292,30 @@ fn toolchain_path(home: Option<String>, toolchain: Option<String>) -> Option<Pat
     })
 }
 
+fn print_rustc_version(args: &[String]) {
+    // make "clippy-driver rustc --version" print rustc --version output
+    if args.len() > 2 && args[1..2] == ["rustc".to_string(), "--version".to_string()] {
+        let rustc_version = Command::new("rustc")
```

Comment:
No, there's no guarantee that there's a `rustc` in the path, or that if there is, it has anything to do with the version of `clippy-driver` we're running.